openoffice.txt

```diff
@@ -1,102 +1,100 @@
 [Section]
 Name = OpenOffice 4.1
-Version = 4.1.5
-License = LGPL
+Version = 4.1.10
+License = AL v2.0
 Description = Open source office suite.
-Size = 134.2 MiB
+Size = 135,98 MiB
```

Comment:
Size is not needed if SizeBytes is present.

Reply:
OK, I just updated the old file...
SizeBytes is preferred?